src/Mouse Position/AstroMath.js

```diff
@@ -0,0 +1,72 @@
+class AstroMath {
+  constructor(targetName) {
+    this.targetName = targetName;
+
+    switch (this.targetName) {
+      case "mars":
+        this.dMajorRadius = 3396190.0;
+        this.dMinorRadius = 3376200.0;
+        break;
+      case "moon":
+        this.dMajorRadius = 1737400.0;
+        this.dMinorRadius = 1737400.0;
+        break;
+      case "mercury":
+        this.dMajorRadius = 2439400.0;
+        this.dMinorRadius = 2439400.0;
+        break;
+      default:
+        console.log("target not supported");
+    }
+  }
+
+  toRadians(degrees) {
+    return (degrees * Math.PI) / 180;
+  }
+
+  toDegrees(radians) {
+    return (radians * 180) / Math.PI;
+  }
+
+  latLonSwitcher(lat, lng, lonTo180, planetocentric, postiveEast, projection) {
+    let convertedLatitude = 0;
+    let convertedLng = lng;
+    let convertedLat = lat;
+    let proj = "";
+
+    if (projection === "EPSG:4326") {
+      proj = "cylindrical";
+    } else {
+      proj = "";
+    }
+
+    if (!lonTo180) {
+      if (proj === "cylindrical") {
+        convertedLng -= 180;
+      }
+      if (convertedLng < 0) {
+        convertedLng += 360;
+      }
+    }
```

Comment:
Why add proj variable if you can check if projection is EPSG:4326 directly instead of adding in a variable and check if that is cylindrical?